packages/wdio-crossbrowsertesting-service/src/launcher.ts

```diff
@@ -6,22 +6,26 @@ import logger from '@wdio/logger'
 
 const log = logger('@wdio/crossbrowsertesting-service')
 
-export default class CrossBrowserTestingLauncher {
-    constructor (options, caps, config) {
-        this.options = options
-        this.config = config
-    }
+export default class CrossBrowserTestingLauncher implements WebdriverIO.ServiceInstance {
+    private _tunnel: boolean = false;
```

Comment:
While we are at it, let's give this property a better name, like `_isUsingTunnel`